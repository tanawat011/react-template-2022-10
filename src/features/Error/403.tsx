import { PATH } from 'Routes'
import { Button } from 'components/Button'

export const Error403 = () => {
  return (
    <div>
      <p>403 Forbidden</p>

      <Button link to={`/${PATH.AUTH.ROOT}`}>
        Back to Login
      </Button>
    </div>
  )
}
