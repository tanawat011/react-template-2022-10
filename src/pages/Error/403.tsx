import { allRoutes } from 'Routes'
import { Button } from 'components/Button'

export const Error403 = () => {
  return (
    <div>
      <p>403 Forbidden</p>

      <Button link to={allRoutes.auth.fullPath}>
        Back to Login
      </Button>
    </div>
  )
}
