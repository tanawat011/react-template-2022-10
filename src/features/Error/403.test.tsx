import { render } from '@testing-library/react'

import { Error403 } from './403'

describe('<Error403 />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<Error403 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
