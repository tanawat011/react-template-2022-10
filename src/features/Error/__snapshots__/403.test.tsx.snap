// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error403 /> match snapshot 1`] = `
<DocumentFragment>
  <div>
    403 Forbidden
  </div>
</DocumentFragment>
`;
